src/sharing/Sharing.component.js

```diff
@@ -1,120 +1,199 @@
-import { PropTypes, createClass, default as React } from 'react';
+/* eslint react/jsx-no-bind: 0 */
```

Comment:
I understand why you did it but lets try to not override the config. There are a few other places where you disable some rules which could have been solve by using different constructs.